src/api/services/lectureService.ts

```
@@ -0,0 +1,486 @@
+    
+    if (assignmentTable) {
+      // Handle multiple tbody elements in assignment table
+      const tbodies = assignmentTable.querySelectorAll('tbody');
+      let assignmentIndex = 0;
+      
+      tbodies.forEach(tbody => {
+        const row = tbody.querySelector('tr');
+        if (row) {
+          const cells = row.querySelectorAll('td');
+          if (cells.length >= 3) {
+            const number = cells[0]?.textContent?.trim();
```

ESLint

[error] 'number' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)